README.md

```diff
@@ -59,22 +59,22 @@ for more details.
 
 ## Requirements
 
-If you have anaconda installed, you can run from the root of this repository:
+If you have Anaconda installed, you can run from the root of this repository:
 
     conda env update -f environment-cpu.yml # if you don't have GPUs
     conda env update -f environment-cuda.yml # if you have GPUs
     conda activate demucs
 
-This will create a `demucs` environmnent with all the dependencies installed.
+This will create a `demucs` environment with all the dependencies installed.
 
 ### Using Windows
 
 If you are using Windows, replace `python3` by `python.exe` in all the commands provided hereafter :)
-Parts of the code are untested on Windows (in particular, training a new model). Please open an issue in case you have a problem.
+Parts of the code is untested on Windows (in particular, training a new model). Please open an issue in case you have a problem.
```

Comment:
"Parts" isn't a word for singular

Reply:
Here the "Parts" represent 'mass noun' I guess, so `is` would be the right word. But I am not sure, what do you suggest @parcerend ?

Reply:
I think "are" was ok here
but tbh, my english is non-native